src/components/VenueList/VenueListSummary.jsx

```diff
@@ -0,0 +1,17 @@
+import {
+  StyledBox,
+  StyledTypography,
+  StyledButton,
+} from './VenueListSummary.styles';
+
+export const VenueListSummary = ({ venues, startIndex, endIndex }) => {
+  return (
+    <StyledBox>
+      <StyledTypography variant="body1">
```

Comment:
This does not match the design. The user should be able to adjust the pagination